test/integration/factory/copy.spec.ts

```diff
@@ -5,59 +5,74 @@ import {
     ASTNodeFactory,
     ASTReader,
     compileJson,
+    CompilerKind,
     compileSol,
     detectCompileErrors
 } from "../../../src";
 
-const cases: Array<[string, string]> = [
+const cases: Array<[string, Array<[CompilerKind, string]>]> = [
     [
         "./test/samples/solidity/declarations/contract_050.json",
-        "./test/samples/solidity/declarations/contract_050.nodes.txt"
+        [
+            [CompilerKind.WASM, "./test/samples/solidity/declarations/contract_050.nodes.wasm.txt"],
+            [
+                CompilerKind.Native,
+                "./test/samples/solidity/declarations/contract_050.nodes.native.txt"
+            ]
+        ]
     ],
-    ["./test/samples/solidity/latest_08.sol", "./test/samples/solidity/latest_08.nodes.txt"]
+    [
+        "./test/samples/solidity/latest_08.sol",
+        [
+            [CompilerKind.WASM, "./test/samples/solidity/latest_08.nodes.wasm.txt"],
+            [CompilerKind.Native, "./test/samples/solidity/latest_08.nodes.native.txt"]
+        ]
+    ]
 ];
 
 describe(`ASTNodeFactory.copy() validation`, () => {
```

Comment:
I feel reasoning for this... But, even if we make a choice we at least have to visually validate the output. Also it helps to actually figure out if output contains valid properties and values. I'm using this test to force myself at least quickly go through the output and ensure that **new nodes** are properly reflected.

However from compiler behavior comparison, it may look like an overkill.

Reply:
Word! Lets keep it then